src/30_Advanced/Joining_Analytics_Sessions.html

```diff
@@ -1,10 +1,16 @@
 <!---{
- "draft": "true"
+ "skipValidation": "true"
```

Comment:
You can use `"hidePreview": true` instead. This will only show the source code listings, but won't add the google analytics tags to the page.

Reply:
Done.